### ISSUE TITLE
Update loading animation icon

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/cozy-sun-bear#155]
- [x] Change the generic loading animation in CSB to a less affective icon.
- [x] Allow for overrides to the loading icon when CSB is deployed in other environments.

Related to [Heliotrope#1750](https://github.com/mlibrary/heliotrope/issues/1750)

**Comment by jmcglone** [comment]
By itself I really like this loader, but in practice, my only nitpick with the animation is that in speedier loads, only a portion of the loader is ever viewed (top quarter), and it isn't clear that what you are seeing is actually a loader. It is just a flash of some graphic and it almost comes off as glitchy. 

Is there a documented way to do the loading animation override that I am missing?

**Comment by respinos** [comment]
@jmcglone - I tweaked the loader so there's always a base faded ring at the start, and sped up the animations.

**Comment by jmcglone** [comment]
@respinos looks great! Sorry it took me forever to review this. Ok to issue a PR with this branch!
